docs/electrical/embedded/styleguide.md

```diff
@@ -1,3 +1,136 @@
 
-*Note that this page is currently an outline and most of the content is not yet written.*
+*Note that this page is currently in an unfinished state and in the process of being written*
 
+## Overview
+This guide follows the general outline and some of the recommendations of [*Barr Group's Embedded C Coding Standard*](https://barrgroup.com/embedded-systems/books/embedded-c-coding-standard), set forth by Michael Barr.
+
+Many of these rules are common sense and you probably are already following them in your own coding style. However, for consistency across the team it is practical to write them explicitly.
+
+??? info "Revision History"
+    - *Revision 0.1 by William K, 11 Jun 2023*
+    - *Revision 0.2 by William K, 12 Aug 2023*
+
+---
+
+## Rules
+
+### General Rules
+#### Line Width
+1. No line width is enforced as we do not do paper code reviews. Please use your best descretion for when a line is considered too long.
```

Comment:
I'd recommend adding a suggested length here (for those new to coding). Either 80, 100, or 120 are common numbers. Not a strict limit. More of an "if you're not sure try to keep below xyz characters"